src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook.go

```diff
@@ -74,6 +85,23 @@ func (r *Cluster) ValidateUpdate(old runtime.Object) error {
 		r.Name, allErrs)
 }
 
+const oneGbInBytes int64 = 1073741824
+
+// ReserveMemoryString is amount of memory that we reserve for other processes than redpanda in the container
+const ReserveMemoryString = "1M"
```

Comment:
this kind of sucks, I would like to have this constant in StatefulsetResource but that's not possible because there will be import cycle, so that's why it's defined here